### PR TITLE
SB-1077 HttpGetRemoteRepositoryCheckStrategy should use connection pool

### PR DESCRIPTION
**Pull request by fuss86**

### CODE REVIEW

File: strongbox-resources/strongbox-storage-api-resources/src/main/resources/etc/conf/strongbox.xml [hunk 1]
```diff
@@ -49,8 +49,7 @@
                     <remote-repository url="https://repo.maven.apache.org/maven2/"
                                        download-remote-indexes="true"
                                        auto-blocking="true"
-                                       checksum-validation="true"
-                                       check-interval-seconds="20" />
```
Comment (fuss86) [inline strongbox-resources/strongbox-storage-api-resources/src/main/resources/etc/conf/strongbox.xml]: @carlspring I think that we should use default 60 seconds interval here too. It was added for test purposes.

File: strongbox-resources/strongbox-storage-api-resources/src/main/resources/etc/conf/strongbox.xml [hunk 2]
```diff
@@ -96,8 +95,7 @@
                                        download-remote-indexes="true"
                                        auto-blocking="true"
                                        checksum-validation="true"
-                                       allows-directory-browsing="false"
-                                       check-interval-seconds="10" />
```
Comment (fuss86) [inline strongbox-resources/strongbox-storage-api-resources/src/main/resources/etc/conf/strongbox.xml]: as above

File: strongbox-resources/strongbox-storage-api-resources/src/main/resources/etc/conf/strongbox.xml (hunk 1 above)
Comment (carlspring) [reply strongbox-resources/strongbox-storage-api-resources/src/main/resources/etc/conf/strongbox.xml]: Okay, I agree.

File: strongbox-resources/strongbox-storage-api-resources/src/main/resources/etc/conf/strongbox.xml (hunk 2 above)
Comment (carlspring) [reply strongbox-resources/strongbox-storage-api-resources/src/main/resources/etc/conf/strongbox.xml]: +1.

**Comment by carlspring** [comment]
Thanks! :)

**Pull request merged**